packages/core/src/controllers/ConnectorController.ts

```diff
@@ -31,6 +31,7 @@ export const ConnectorController = {
   },
 
   setConnectors(connectors: ConnectorControllerState['connectors']) {
+    connectors.forEach(this.syncIfAuthConnector)
```

Comment:
Why do we need to do this? When we switch to multiple adapters, this will run the `syncIfAuthConnector` function for every connector we have. Is it okay? What about if the `wagmi/ethers/ethers5`'s auth connector? It'll pass the if clause right?ç